### ISSUE TITLE
Add support for initial context to evaluate commands

### ISSUE THREAD

**Issue by klinvill** [alygin/vscode-tlaplus#213]
[Original observation](https://github.com/alygin/vscode-tlaplus/issues/207#issuecomment-845354312) from @lemmy:

>I realized that the dialog could be used to support evaluating expressions in a spec that declares variables and/or constants, but lacks a model:
>
>```tla
>---------------------- MODULE ATD ---------------------
>EXTENDS Naturals
>
>CONSTANT N
>
>ASSUME NIsPosNat == N \in Nat \ {0}
>
>Node == 0 .. N-1
>=============================================================================
>```
>
>If a user tries to evaluate `Node`, one gets:
>
>```
>Error evaluating expression:
>The constant parameter N is not assigned a value by the configuration file.
>```
>
>It would be nice if one could type `WITH N <- 42` in the prompt (suffix of regular TLA+ expression `INSTANCE ATD WITH N <- 42`).

**Comment by klinvill** [comment]
While using the prompt would be nice for cases where you only have a few constants (like above), it might get unwieldy for specs with many constants. This could be partly improved by persisting previous assignments at a project level. Would it be better to supply initial context by reading in constants from a config file?

**Comment by lemmy** [comment]
We already have a config file (YourSpec.cfg), and I don't think TLA+ values should be persisted as project-level VSCode settings.

**Comment by klinvill** [comment]
I agree, I think it makes sense to take advantage of a more complete config like the MC approach you mention. In that case, I'd vote for deferring this until after issue 159 is resolved
